.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: Mark and close stale pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-stale: 14
+        days-before-close: 7
+        stale-pr-message: |
+          Hello! Thank you for this proposed change to cloud-init. This pull request hasn't seen any activity in 14 days, so is being marked as stale. This pull request will be automatically closed if no activity occurs within the next 7 days.
```

Comment:
```Hello! Thank you for this proposed change to cloud-init. This pull request is now marked as stale as it has not seen any activity in 14 days. If no activity occurs within the next 7 days, this pull request will automatically close.``

No contractions and avoid the passive.

Reply:
Done.

Reply:
@OddBloke  Please add something like.
If the pull request gets closed, you are welcome to re-open the pull request when review comments have been addressed or responded to.